OPERATING.md

```diff
@@ -171,6 +171,11 @@ In the diagram above there is a single broker for all queues, but the adaptor su
 An example daisy chaining environment is provided in [/test-suite/daisy-chaining/](/test-suite/daisy-chaining),
 and each environment variable described within [Inbound message queue variables](#inbound-message-queue-variables).
 
+### Retrying and dead-letter queue
+The adaptor will put messages it doesn't recognise into dead letter queue. Messages that can't be delievered due to different
+exceptions and which exceeded the threshold of retries will also be sent to the dead letter queue.
+The number of retries is configured in application.yml file. If the value of retries is not specified then, by default, it will be set to 3.
```

Comment:
End users have no way to modify the `application.yml` file directly, as it's bundled up as a Docker image, they can only configure the app via the environment variables.